src/helpers/utils.tsx

```diff
@@ -398,17 +398,21 @@ export function roundToPrecision(n: number, precision: number = 0): number {
   return Math.round(n * factor) / factor;
 }
 
-export function stopPropagation(e: Event | MouseEvent | any) {
-  e.stopPropagation();
-}
-
+/** handler that cancels an event's default behavior
+ *
+ * @param {Event | MouseEvent} e - a synthetic event wrapper around native dom events
+ */
 export function preventDefault(e: Event | MouseEvent | any) {
```

Comment:
It seems that this one is not used anywhere?

Reply:
yeah the idea was to only leave the one that has them both as a standalone function.
Dont think i was sober enough during this pr